app/src/main/java/com/yuneec/example/component/fragment/CameraSettingsFragment.java

```diff
@@ -53,6 +58,12 @@
 
     Camera.ShutterSpeedS shutterSpeedS = new Camera.ShutterSpeedS();
 
+    Button capturePicture;
+
+    Button video;
+
+    private Camera.Mode cameraMode = Camera.Mode.PHOTO;
```

Comment:
Do you know how it is initialized? What about starting with `UNKNOWN`?

Reply:
Yes makes sense. Will change it.